x-pack/test/functional/apps/ml/data_visualizer/files_to_import/artificial_server_log

```diff
@@ -1,19 +1,20 @@
-2018-01-06 16:56:14.295748 INFO host:'Server A' Incoming connection from ip 123.456.789.0
-2018-01-06 16:56:15.295748 INFO host:'Server A' Incoming connection from ip 123.456.789.1
-2018-01-06 16:56:16.295748 INFO host:'Server A' Incoming connection from ip 123.456.789.2
-2018-01-06 16:56:17.295748 INFO host:'Server A' Incoming connection from ip 123.456.789.3
-2018-01-06 16:56:18.295748 INFO host:'Server B' Incoming connection from ip 123.456.789.0
-2018-01-06 16:56:19.295748 INFO host:'Server B' Incoming connection from ip 123.456.789.2
-2018-01-06 16:56:20.295748 INFO host:'Server B' Incoming connection from ip 123.456.789.3
-2018-01-06 16:56:21.295748 INFO host:'Server B' Incoming connection from ip 123.456.789.4
-2018-01-06 16:56:22.295748 WARN host:'Server A' Disk watermark 80%
-2018-01-06 17:16:23.295748 WARN host:'Server A' Disk watermark 90%
-2018-01-06 17:36:10.295748 ERROR host:'Server A' Main process crashed
-2018-01-06 17:36:14.295748 INFO host:'Server A' Connection from ip 123.456.789.0 closed
-2018-01-06 17:36:15.295748 INFO host:'Server A' Connection from ip 123.456.789.1 closed
-2018-01-06 17:36:16.295748 INFO host:'Server A' Connection from ip 123.456.789.2 closed
-2018-01-06 17:36:17.295748 INFO host:'Server A' Connection from ip 123.456.789.3 closed
-2018-01-06 17:46:11.295748 INFO host:'Server B' Some special characters °!"§$%&/()=?`'^²³{[]}\+*~#'-_.:,;µ|<>äöüß
```

Comment:
If possible, I'd like to keep some special characters in the log sample that we're loading, just to make sure it loads ok.